adapter-mongodb/src/main/resources/activities/mongodb-basic.yaml

```diff
@@ -2,49 +2,45 @@ description: An example of a basic mongo insert and find.
 
 scenarios:
   default:
-    rampup: run driver=mongodb tags==block:rampup cycles===TEMPLATE(rampup-cycles,1000000) threads=auto
-    main: run driver=mongodb tags==block:"main.*" cycles===TEMPLATE(main-cycles,10000000) threads=auto
+    rampup: run driver=mongodb tags==block:"rampup.*", cycles===TEMPLATE(rampup-cycles,10) threads=auto errors=timer,warn
+    write: run driver=mongodb tags==block:"write.*", cycles===TEMPLATE(write-cycles,10) threads=auto errors=timer,warn
+    read: run driver=mongodb tags==block:"read.*", cycles===TEMPLATE(read-cycles,10) threads=auto errors=timer,warn
 
 bindings:
-  seq_key: Mod(<<keyCount:1000000>>L); ToInt()
-  seq_value: Mod(<<valueCount:1000000000>>L); <<valueSizeDist:Hash()>>; ToString() -> String
-  rw_key: <<keyDist:Uniform(0,1000000)->long>>; ToInt()
-  rw_value: <<valDist:Uniform(0,1000000000)->int>>; <<valueSizeDist:Hash()>>; ToString() -> String
+  seq_key: Mod(<<keyCount:1000000>>); ToString() -> String
```

Comment:
All versions of *.ToString return a String, so this output type qualifier is not explicitly required. This could confuse users who think it is.